### PR TITLE
Added sygvj

### PR DESCRIPTION
**Pull request by tfalders**
Adds sygvj to rocSOLVER.

### CODE REVIEW

**Review by jzuniga-amd — APPROVED** [review]
LGTM

File: library/include/rocsolver/rocsolver-functions.h [hunk 1]
```diff
@@ -17915,6 +17915,788 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_zhegvd_strided_batched(rocblas_handle
...
+    \f[
+        \begin{array}{cl}
+        A X = \lambda B X & \: \text{1st form,}\\
+        A B X = \lambda X & \: \text{2nd form, or}\\
+        B A X = \lambda X & \: \text{3rd form,}
+        \end{array}
+    \f]
+
+    depending on the value of itype. The eigenvalues are found using the iterative
+    Jacobi algorithm, and are returned in ascending order.The eigenvectors are computed
```
Comment (cgmb) [inline library/include/rocsolver/rocsolver-functions.h:17934]: There's a space missing after "order."

File: library/include/rocsolver/rocsolver-functions.h [hunk 2]
```diff
@@ -17915,6 +17915,788 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_zhegvd_strided_batched(rocblas_handle
...
```
Comment (cgmb) [inline library/include/rocsolver/rocsolver-functions.h:18053]: .

File: library/include/rocsolver/rocsolver-functions.h [hunk 3]
```diff
@@ -17915,6 +17915,788 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_zhegvd_strided_batched(rocblas_handle
...
+    \f[
+        \begin{array}{cl}
+        A_j X_j = \lambda B_j X_j & \: \text{1st form,}\\
+        A_j B_j X_j = \lambda X_j & \: \text{2nd form, or}\\
+        B_j A_j X_j = \lambda X_j & \: \text{3rd form,}
+        \end{array}
+    \f]
+
+    depending on the value of itype. The eigenvalues are found using the iterative
+    Jacobi algorithm, and are returned in ascending order.The eigenvectors are computed
```
Comment (cgmb) [inline library/include/rocsolver/rocsolver-functions.h:18172]: .

File: library/include/rocsolver/rocsolver-functions.h [hunk 4]
```diff
@@ -17915,6 +17915,788 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_zhegvd_strided_batched(rocblas_handle
...
```
Comment (cgmb) [inline library/include/rocsolver/rocsolver-functions.h:18302]: .

File: library/include/rocsolver/rocsolver-functions.h [hunk 5]
```diff
@@ -17915,6 +17915,788 @@ ROCSOLVER_EXPORT rocblas_status rocsolver_zhegvd_strided_batched(rocblas_handle
...
```
Comment (cgmb) [inline library/include/rocsolver/rocsolver-functions.h:18432]: .

**Comment by cgmb** [comment]
@tfalders, the CI is failing because there's no suitable rocBLAS package available. You could wait for a new rocBLAS package and re-run the rocSOLVER CI jobs to get all the checks to pass, but that will probably take until tomorrow. That's the best-case scenario, as there's no guarantee there won't be new errors stemming from rocBLAS or the CI configuration at that time. Feel free to override the checks to merge today.

@saadrahim, to make 'Required' checks more useful, we need to eliminate this class of test failures.

**Pull request merged**